packages/js/product-editor/src/components/variations-table/variation-actions-menus/variation-quick-update-menu-item.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import { Slot, Fill, MenuItem } from '@wordpress/components';
+import { createElement, Fragment } from '@wordpress/element';
+import {
+	createOrderedChildren,
+	sortFillsByOrder,
+} from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { MenuItemProps } from './types';
+import {
+	MULTIPLE_UPDATE,
+	SINGLE_UPDATE,
+	VARIATION_ACTIONS_SLOT_NAME,
+} from './constants';
+
+const DEFAULT_ORDER = 20;
+
+export const getGroupName = (
+	group?: string,
+	isMultipleSelection?: boolean
+) => {
+	const nameSuffix = isMultipleSelection
+		? `_${ MULTIPLE_UPDATE }`
+		: `_${ SINGLE_UPDATE }`;
+	return group
+		? `${ VARIATION_ACTIONS_SLOT_NAME }_${ group }${ nameSuffix }`
+		: VARIATION_ACTIONS_SLOT_NAME;
+};
+
+export const getMenuItem = (
+	children: React.ReactNode,
+	onClick: () => void
+) => <MenuItem onClick={ onClick }>{ children }</MenuItem>;
```

Comment:
We need to pass through the `selection` and `onChange` so that fills can access that in their `onClick` handler and actually do something!

We should also pass  the`onClose` callbacks, as that will be needed too for extensions to close the menu when they have done their action!